API.pm

```diff
@@ -643,20 +643,19 @@ sub _get {
 	
 	if (main::DEBUGLOG && $log->is_debug) {
 		my $data = $url;
-		$data =~ s/(?:$aid|$token)//g;
+		#$data =~ s/(?:$aid|$token)//g; don't remove the api key.
 		$log->debug($data);
 	}
 	
 	if ($params->{_wipecache}) {
 		$cache->remove($url);
 	}
-	
-	if (!$params->{_nocache} && (my $cached = $cache->get($url))) {
-		main::DEBUGLOG && $log->is_debug && $log->debug("found cached response: " . Data::Dump::dump($cached));
-		$cb->($cached);
-		return;
+	# don't use cached 
+	#if (!$params->{_nocache} && (my $cached = $cache->get($url))) {
+	#	main::DEBUGLOG && $log->is_debug && $log->debug("found cached response: " . Data::Dump::dump($cached));
+	#	$cb->($cached);
+	#	return;
 	}
```

Comment:
Both these features have been implemented for a reason. If you start _not_ to cache, we all risk to get banned by them for violating their terms of use. What problem are you trying to solve with this?